requirements/edx/base.txt

```diff
@@ -51,7 +51,9 @@ edx-lint==0.4.3
 astroid==1.3.8
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.1.1
-edx-enterprise==0.19.1
+# enterprise app with desired changes
+-e git+https://github.com/edx/edx-enterprise.git@e0be788a74fbbce8be04a75b2ddca8843db17426#egg=edx-enterprise
```

Comment:
No its not temporary. I planing to create a separate PR with changes required in edx-enterprise after first round of review of this PR and we should be merging both of these PRs at the same time.